### PR TITLE
batches: update changelog

### PR DESCRIPTION
**Pull request by BolajiOlajide**
## Test plan

Update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ All notable changes to Sourcegraph are documented in this file.
 - Added Azure DevOps Services as a Tier 1 Code Host, including: repository syncing, permissions syncing, and Batch Changes support. [#46265](https://github.com/sourcegraph/sourcegraph/issues/46265)
 - Added feature to disable some fields on user profiles for SCIM-controlled users. [#48816](https://github.com/sourcegraph/sourcegraph/pull/48816)
 - Native support for ingesting and searching GitHub topics with `repo:has.topic()` [#48875](https://github.com/sourcegraph/sourcegraph/pull/48875)
+- Role-based Access Control is now available as an experimental enterprise feature. It is currently only supported for Batch Changes functionality. [#43276](https://github.com/sourcegraph/sourcegraph/issues/43276)
+- Creation of batch changes can now be restricted for users. This is made possible by the Role-based Access Control implementaiton. [#34491](https://github.com/sourcegraph/sourcegraph/issues/34491)
```
Comment (eseliger) [inline CHANGELOG.md:37]:
```suggestion
- Creation of batch changes can now be restricted for users. This is made possible by the Role-based Access Control implementation. [#34491](https://github.com/sourcegraph/sourcegraph/issues/34491)
```

**Review by eseliger — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (BolajiOlajide) [reply CHANGELOG.md:37]: Duh! 🤦🏽 

Thanks.

**Comment by courier-new** [comment]
Thanks for this! Maybe, let's hold off on merging until we've written our docs, too, so that we can link to them from the entries. 😄

**Comment by courier-new** [comment]
I'm going to tweak the wording a bit and add one for outbound webhooks, too. 😄

**Comment by coury-clark** [comment]
this backport is still open, should we merge / close?

**Comment by BolajiOlajide** [comment]
I'll make an update and then merge soon. Need to link some of the new docs we added for RBAC and Outgoing webhooks here.

**Review by courier-new — APPROVED** [review]

**Pull request merged**